generatorreceiver/internal/flags/flag_embed.go

```diff
@@ -26,6 +29,21 @@ func (f *EmbeddedFlags) IsDefault() bool {
 	return f.FlagSet == "" && f.FlagUnset == ""
 }
 
+func (f *EmbeddedFlags) UpdatedTime() time.Time {
+	if f.FlagSet != "" {
+		if set := Manager.GetFlag(f.FlagSet); set != nil {
```

Comment:
`set` (and `unset` below) should never be `nil` here since we perform validation on all flags first, but no harm in keeping this

Reply:
Nice, I've update both this and the `ShouldGenerate()` function to drop this validation and simplify logic.